_data/releases.tsv

```diff
@@ -45,3 +45,5 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2020-10-06	prod89	0.8.9	14424	1286147		9903464	70616509	186.92806861126402	24.170604	433.145737887
 2020-11-19	prod90	0.9.0	15677	1746563		10646676	72846145	192.92555930017602	26.418725	472.513680031
 2020-12-10	prod91	0.9.1	15696	1753475		10708889	73157773	196.00307508509002	26.792054	476.26
+2021-01-26	prod92	0.9.2	17211	1753475	11423912	76708906	215.07339019756503	28.162387	520.03
```

Comment:
If you preview this file GitHub says there's a mismatch in the number of columns:
https://github.com/IDR/idr.openmicroscopy.org/blob/184f1eb72e76e825bc78cc64d8780a45c29704d6/_data/releases.tsv

![image](https://user-images.githubusercontent.com/1644105/105826420-9f875600-5fb8-11eb-87dc-20aafebe187b.png)

If it's correct you should see a formatted table as with https://github.com/IDR/idr.openmicroscopy.org/blob/184f1eb72e76e825bc78cc64d8780a45c29704d6/_data/studies.tsv